ops/ccs-ops-misc/cert-checker/env.sample

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 
 # endpoints to check
-export DEFAULT_ENDPOINTS="google.com:443 bing.com:443"
+export DEFAULT_ENDPOINTS="foo:443 bar:443"
 
 # notifications
-export SEND_NOTIFICATIONS="false"     # Send slacks/pages? "true" or "false"
-export WARN_WHEN="3000:4500"              # WARN when expiration is >= 30 but less than 45 days
-export ALERT_WHEN="1500:3000"             # ALERT when expiration is >= 15 but less than 30 days
-export PAGE_WHEN="1:1500"               # PAGE when expiration >= 0 but less than 15
+export SEND_NOTIFICATIONS="false"   # Send slacks/pages? "true" or "false"
+export WARN_WHEN="3000:4500"        # WARN  when >= 30 and < 45 days
+export ALERT_WHEN="1500:3000"       # ALERT when >= 15 and < 30 days
+export PAGE_WHEN="1500"             # PAGE  when <= 15 days
```

Comment:
these seem like they should be 30, 45, etc and not 3000, 4500, etc. ?

Reply:
Fixed